Rakefile

```diff
@@ -1,5 +1,27 @@
 $LOAD_PATH.unshift '.'
 
+# Create our application's ESPs
+def processors(db_connection, tracker)
```

Comment:
Left these as params so the disconnect call in the `run_processors` looks less magical.